### PR TITLE
fix(sqlsmith): use simple query interface

### PR DESCRIPTION
**Pull request by kwannoel**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

- Workaround #7949 by using simple query mode.
- Use named `BIGINT` literal to avoid ambiguity.

https://www.postgresql.org/docs/current/protocol-flow.html#id-1.10.6.7.4

Screenshot 2023-02-20 at 11 28 26 AM

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
- [x] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

**Comment by jon-chuang** [comment]
This is a quick fix. Let's fix https://github.com/risingwavelabs/risingwave/issues/7949 in the long term, unclear if it is a priority for now, but definitely worried for e.g. BI tools.

File: src/tests/sqlsmith/src/sql_gen/scalar.rs [hunk 1]
```diff
@@ -41,9 +41,10 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
         // e.g. -1 becomes -(1).
         // See: https://github.com/risingwavelabs/risingwave/issues/4344
         match *typ {
-            T::Int64 => Expr::Nested(Box::new(Expr::Value(Value::Number(
-                self.gen_int(i64::MIN as isize, i64::MAX as isize),
-            )))),
+            T::Int64 => Expr::Nested(Box::new(Expr::TypedString {
```
Comment (jon-chuang) [inline src/tests/sqlsmith/src/sql_gen/scalar.rs:44]: Does this mean that the query will look like
`select 10213535::bigint;` instead of `select 10213535`?

**Review by jon-chuang — APPROVED** [review]
Rest LGTM

File: src/tests/sqlsmith/src/sql_gen/scalar.rs (hunk 1 above)
Comment (kwannoel) [reply src/tests/sqlsmith/src/sql_gen/scalar.rs:44]: select bigint '1';

**Pull request merged**